charts/invenio/templates/_helpers.tpl

```diff
@@ -334,7 +334,7 @@ INVENIO_SEARCH_HOSTS: {{ printf "[{'host': '%s'}]" (include "invenio.opensearch.
 INVENIO_SITE_HOSTNAME: '{{ include "invenio.hostname" $ }}'
 INVENIO_SITE_UI_URL: 'https://{{ include "invenio.hostname" $ }}'
 INVENIO_SITE_API_URL: 'https://{{ include "invenio.hostname" $ }}/api'
-INVENIO_DATACITE_ENABLED: "False"
+INVENIO_DATACITE_ENABLED: {{ternary "True" "False" .Values.invenio.datacite.enabled | quote }}
```

Comment:
We need this to be a string, hence the `quote`. Otherwise, it will get translated into `true` and the final value would be a string inside Invenio:
```
In [1]: app.config['DATACITE_ENABLED']
Out[1]: 'true'
```